infra/README.md

```diff
@@ -0,0 +1,58 @@
+
+# OpenSearch cluster to test the interceptor at development time
+
+This project creates a simple OpenSearch cluster for development purposes. Its main objective is to enable collaborators to easily test features against real cluster.
+
+To manually create a virtualenv and install the dependencies on MacOS and Linux:
+
+```
+make .venv
+```
+
+After the init process completes and the virtualenv is created, you can use the following step to activate your virtualenv.
+
+```
+source .venv/bin/activate
+```
```

Comment:
Consider using [Pipenv](https://pipenv.pypa.io/en/latest/) as an alternative, IMO it's a lot simpler and behaves like other Rubygems/npm package managers.

Reply:
As this very simple project I prefer to leave it like this. I don't need to install the extra package and the version is updated by `apt-get`. I also have the impression that Python version management world is quite divided and opinionated, I don't care much, I'm happy to make the change in the future if we find it to be more convenient but I'll carry on as is this time.